src/main/app/src/components/SignUp.js

```diff
@@ -0,0 +1,100 @@
+import { Button, Grid, TextField, makeStyles, Paper, useMediaQuery, useTheme } from '@material-ui/core';
+import axios from 'axios';
+import React, { useState } from 'react';
+
+
+//TODO Make Inline Styling
+
+const useStyles = makeStyles(theme => ({
+  divStyle: {
+    alignContent: 'centre',
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: 400,
+    minWidth: 30,
+    borderRadius: 20,
+    border: '30px solid',
+    borderImage: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+    padding: 50,
+    borderImageSlice: 1
+  }
+}));
+
+const postMe = async data => {
+  try {
+    const api = '/api/signup/';
+    const posted = await axios.post(api, data);
+    console.log(`STATUS CODE: ${posted.status}`);
+    console.log(`DATA: ${posted.data || "Nothing"}`);
+    return posted.status == 200 ? posted.data || {} : {};
```

Comment:
Use `===` instead of `==`, here it is not a big deal but in general I would try to avoid `==` in JavaScript and go with `===` instead.

The difference is that `==` will perform type coercion and `===` will not. You almost always would rather not do type coercion when testing equality, so `===` is best practice to use everywhere you can.